packages/okta-react/README.md

```diff
@@ -21,7 +41,7 @@ This library currently supports:
 - [React Quickstart](https://facebook.github.io/react/docs/installation.html#creating-a-new-application)
   - If you don't have a React app, or are new to React, please start with this guide. It will walk you through the creation of a React app, creating routes, and other application development essentials.
 - [Okta Sample Application](https://github.com/okta/samples-js-react)
-  - A fully functional sample application.
+  - A fully functional sample application built using this SDK.
 - [Okta React Quickstart](https://developer.okta.com/quickstart/#/react/nodejs/express)
```

Comment:
Let's link to the new guide instead: https://developer.okta.com/docs/guides/sign-into-spa/react/before-you-begin/